test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,32 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
+        // no item
+        assertNotNull();
+
+        // only one item
+        assertNull((Object) null);
+        assertNotNull("abc");
+        assertNotNull(1);
+        assertNotNull("");
+
+        // multiple items
```

Comment:
To make the test cases clearer, maybe you can try further separating the case into `// multiple items with null` & `// multiple items without null`.